source/schema.txt

```diff
@@ -338,3 +338,424 @@ To delete a location filter from the map:
 .. |query-build-ref| replace:: :ref:`build-query`
 
 .. |query-build-type| replace:: :ref:`build-query`
+
+Export Your Schema
+------------------
+
+You can export your schema after analyzing it. This is useful for 
+sharing your schema and comparing schemas across collections.
+
+If you have not already done so, analyze your schema:
+
+1. Select your desired collection and click the :guilabel:`Schema` tab.
+
+#. Click :guilabel:`Analyze Schema`.
+
+.. note::
+   When |compass-short| analyzes your schema, it samples a random 
+   subset of documents from your collection. To learn more about 
+   sampling, see :ref:`Sampling <sampling>`.
+
+Once your schema has been analyzed, export your schema:
+
+1. In the top menu bar, click :guilabel:`Collection`.
+
+#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+
+.. figure:: /images/compass/share-schema-json.png
+   :alt: Image showing Collection dropdown
+
+Your schema is copied to your clipboard as a |json| object.
+
+Schema Object Properties
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Schema objects have properties ``count`` and ``fields``:
+
+- ``count`` is an integer that represents the number of documents 
+  :ref:`sampled <sampling>` from the collection to generate the schema.
+
+- ``fields`` is an array of ``n`` metadata objects that represent each 
+  field in the documents analyzed during sampling. Each element in the 
+  ``fields`` array contains:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 25 35 40
+
+  * - Field name
+    - Data type
+    - Description
+
+  * - ``name``
+    - String
+    - Name of ``fields[n]``, e.g. ``_id``.
+
+  * - ``path``
+    - String
+    - Path to ``fields[n]`` within a document.
+    
+  * - ``count``
+    - Integer
+    - The number of documents that ``fields[n]`` appears on.
+
+  * - ``types``
+    - Array
+    - List of ``n`` metadata objects that represent each data type that 
+      appears in ``fields[n]``.
```

Comment:
Consideration: I know we've gone back and forth on this but I'm wondering if "appears in the current `fields` array element" is maybe clearer. Up to you.

Reply:
I agree this has to be changed again across the board - ``fields[n]`` is an object, so using it to refer to the field we're trying to talk about will be confusing to someone. I don't think we can say "current fields array element" for the same reason. It's more like "the field in your documents that ``fields[n]`` describes." Brainstorming more.

Reply:
I might have overthought this. Going with "this field," I think it's intuitive and reads well.

Changing the table heading to "property" instead of "field name" to cut down on using "field."